app/ai/voice/agents/breeze_buddy/agent.py

```
@@ -114,6 +115,7 @@ def __init__(
         self.root_span = (
             None  # Store OpenTelemetry span reference for updating with evaluation data
         )
+        self._pipeline_tracing: Optional[PipelineTracing] = None
```

Ruff (0.14.10)

Undefined name `PipelineTracing`

(F821)